test/N3Parser-test.js

```diff
@@ -975,14 +1126,59 @@ describe('Parser', () => {
       shouldParse('<d> <e> <<<<<a> <b> <c>>> <f> <g>>>.',
         ['d', 'e', [['a', 'b', 'c'], 'f', 'g']]));
 
+    it('should not parse compound blank node inside quoted triple subject',
+      shouldNotParse('<< [ <x> <y> ] <a> <b> >> <c> <d>.',
+        'Compound blank node expressions not permitted within quoted triple on line 1.'
+    ));
+
+    it('should not parse compound blank node inside quoted triple predicate',
+      shouldNotParse('<< <a> [ <x> <y> ] <b> >> <c> <d>.',
+        'Disallowed blank node as predicate on line 1.'
+    ));
+
+    it('should not parse compound blank node inside quoted triple object',
+      shouldNotParse('<< <a> <b> [ <x> <y> ] >> <c> <d>.',
+        'Compound blank node expressions not permitted within quoted triple on line 1.'
+    ));
+
+    it('should not parse empty list inside quoted triple subject',
+      shouldNotParse('<< () <a> <b> >> <c> <d>.',
+        'Unexpected list inside quoted triple on line 1.'
+      ));
+
+    it('should not parse non-empty list inside quoted triple subject',
+      shouldNotParse('<< ( <f> ) <a> <b> >> <c> <d>.',
+        'Unexpected list inside quoted triple on line 1.'
+      ));
+
+    it('should not parse empty list inside quoted triple predicate',
+      shouldNotParse('<< <a> () <b> >> <c> <d>.',
+        'Expected entity but got ( on line 1.'
+      ));
+
+    it('should not parse non-empty list inside quoted triple predicate',
+      shouldNotParse('<< <a> ( <f> ) <b> >> <c> <d>.',
+        'Expected entity but got ( on line 1.'
+    ));
+
+    it('should not parse empty list inside quoted triple object',
+      shouldNotParse('<< <a> <b> () >> <c> <d>.',
+        'Unexpected list inside quoted triple on line 1.'
+    ));
+
+    it('should not parse non-empty list inside quoted triple object',
+      shouldNotParse('<< <a> <b> ( <f> ) >> <c> <d>.',
+        'Unexpected list inside quoted triple on line 1.'
+    ));
+
     it('should not parse nested RDF* statements that are partially closed',
```

Comment:
Do you really mean the deprecated `RDF*`, or should this name the current `RDF-star`?
```suggestion
    it('should not parse nested RDF-star statements that are partially closed',
```
This comment applies to this whole file...